CONTRIBUTING.adoc

```diff
@@ -0,0 +1,57 @@
+= Contributing
+
+In the spirit of free software, _everyone_ is encouraged to help improve this project.
+
+To contribute code, simply fork the project on GitHub, hack away and send a pull request with your proposed changes.
+
+Feel free to use the https://github.com/asciidoctor/atom-language-asciidoc/issues[issue tracker] to provide feedback or suggestions in other ways.
+
+== Development
+*todo*
+
+=== Retrieve the source code
+*todo*
+
+=== Prepare for business
+*todo*
+
+== Quality assurance
+*todo*
+
+=== CoffeeScript lint
+This package uses http://www.coffeelint.org/[CoffeeLint] to ensure that the code is uniformly structured.
+The linter definitions are defined in the link:coffeelint.json[coffeelint.json].
+
+When developing in Atom, the https://atom.io/packages/linter-coffeelint[linter-coffeelint] package exposes the linting in your editor.
+It does require CoffeeLint to be http://www.coffeelint.org/#install[installed].
+
+CoffeeLint can also be triggered from the commandline:
+
+`coffeelint -f coffeelint.json grammars/asciidoc.cson`
```

Comment:
Although I'd prefer the first option for cleanness, it happily parses the `node_modules` directory on my machine. And as expected this triggers a load of errors. The latter is better of that reason.